backend/src/main/resources/application-local.yml

```diff
@@ -29,13 +30,15 @@ spring:
     url: jdbc:postgresql://localhost:54321/teambalance
   devtools:
     add-properties: true
-  jpa:
-    show-sql: true
-
   security:
     user:
       password: admin
       name: admin
       roles:
         - admin
         - user
+
+logging:
+  level:
+#    org.jooq: debug
+    org.jooq: info
```

Comment:
Keep it at debug locally? Makes it at least apparent some more thought needs to go into query optimisation :O